analyzere/__init__.py

```diff
@@ -2,6 +2,7 @@
 password = ''
 base_url = 'http://localhost:8000/'
 upload_poll_interval = 0.1
+upload_chunk_size = 16 * (1 << 20)
```

Comment:
Ok, will fix. I find it more convenient to type and remember:
- `1 << 10 == 1024` aka. kibi
- `1 << 20 == 1024 * 1024` aka. mebi
- `1 << 30 == 1024 * 1024 * 1024` aka. gibi

... and so on.